### PR TITLE
Acrolinx for Python

### PR DESCRIPTION
**Pull request by adrianhall**

### CODE REVIEW

File: docs/design/python/sections/PythonCodingGuidelines.mdk [hunk 1]
```diff
@@ -199,11 +201,11 @@ foo(1, 2, 3)
 ## Public vs "private"
 
 ~ Must {#python-single-underscore}
-use a single leading underscore to indicate that a name is not part of the public API and is not guaranteed stable.
+use a single leading underscore to indicate that a name isn't part of the public API.  Non-public APIs aren't guaranteed to be stable.
 ~
 
 ~ MustNot {#python-no-leading-underscore}
-use leading double underscore prefixed method names unless name clashes in inheritence hierarchy are likely. This is rare. 
+use leading double underscore prefixed method names unless name clashes in the inheritance hierarchy are likely.
```
Comment (johanste) [inline docs/design/python/sections/PythonCodingGuidelines.mdk]: I do want to indicate that this is the exception to the rule. Many developers coming from a more strongly typed background greatly overuse leading double underscores. This phrasing loses that emphasis.

File: docs/design/python/sections/PythonCodingGuidelines.mdk [hunk 2]
```diff
@@ -254,23 +256,23 @@ Examples: [`asyncio.loop.call_soon_threadsafe`](https://docs.python.org/3/librar
 ~
 
 ~ Should
-allow callers to pass in an [`Executor`](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Executor) instance rather than defining your own thread or process management for parallelism. 
+allow callers to pass in an [`Executor`](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Executor) instance rather than defining your own thread or process management for parallelism.
 
-An exception to this rule is if your library is using a background thread to wait on blocking I/O with only occasional processing, and it is not exposed to the caller in any way. An example of this is the background poller thread for the default `LROPoller` implementation.
+You may do your own process management if the thread isn't exposed to the caller in any way. For example, the `LROPoller` implementation uses a background poller thread.
```
Comment (johanste) [inline docs/design/python/sections/PythonCodingGuidelines.mdk]: process management => thread management?

File: docs/design/python/sections/Dependencies.mdk [hunk 3]
```diff
@@ -3,27 +3,29 @@
...
-use external dependencies outside the list of [well known dependencies](#python-dependency-well-known). To get a new dependency added, contact @adparch. 
+use external dependencies outside the list of [well known dependencies](#python-dependency-well-known). To get a new dependency added, contact [adparch](https://github.com/azure/azure-sdk/issues). 
 ~
 
 ~ MustNot {#python-no-vendor-deps}
-vendor dependencies unless approved by @adparch. Vendoring can sometimes be preferred over taking a dependency of an external package. @adparch can help determine if this is appropriate.
+vendor dependencies unless approved by [adparch](https://github.com/azure/azure-sdk/issues).
 
-*Vendoring dependencies in Python means including the source from another package as if it was part of your package*
+When you vendor a dependency in Python, you include the source from another package as if it was part of your package. Copying the source is sometimes preferable over taking a dependency.
```
[the remark this answers is not in the document]
Comment (adrianhall) [reply docs/design/python/sections/Dependencies.mdk]: Perhaps replace the final sentence with "Engage with the [python architect](github...) prior to considering including the source from another package."

File: docs/design/python/sections/Logging.mdk [hunk 4]
```diff
@@ -4,49 +4,53 @@ use Pythons standard [logging module](https://docs.python.org/3/library/logging.
...
 ~ Must {#python-logging-levels-debug}
-use the `DEBUG` logging level for detailed trouble shooting scenarios. This is primarily intended for developers or system administrators to diagnose specific failures.
+use the `DEBUG` logging level for detailed trouble shooting scenarios.
+
+The `DEBUG` logging level is intended for developers or system administrators to diagnose specific failures.
 ~
 
 ~ MustNot {#python-logging-no-sensitive-info}
-send sensitive information in log levels other than `DEBUG`, e.g. remove account keys when logging headers.
+send personally identifiable or sensitive information in log levels other than `DEBUG`.  For example, redact or remove account keys when logging headers.
```
[the remark this answers is not in the document]
Comment (adrianhall) [reply docs/design/python/sections/Logging.mdk]: This one probably needs a discussion between the two of us.

**Pull request merged**